modules/platforms/dotnet/Apache.Ignite.Core/Impl/Binary/BinaryUtils.cs

```diff
@@ -69,7 +69,7 @@ internal static class BinaryUtils
         public const int ObjTypeId = -1;
 
         /** Ticks for Java epoch. */
-        private static readonly long JavaDateTicks = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc).Ticks;
+        public static readonly long JavaDateTicks = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc).Ticks;
```

Comment:
Let's keep this encapsulated and create a method to convert Java timestamp to DateTime.